intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -128,19 +129,29 @@ We might say...
 
 ### State and Behavior
 
-When we start defining our own classes, we will specify their **state** and their generic **behavior.**
+When we start defining our own classes, we will specify two main things: their **state** and their **behavior**.
+
+In the context of Object-Oriented Programming, **state** is used to refer to all the variables packaged within a class and **behavior** is used to refer to all the functions packaged within a class.
+
+### Attributes and Methods
+
+There are two more words you will often see in the context of OOP: **attribute** and **method**.
+
+For whatever reason, engineers long-ago decided we needed new words to describe variables and functions that make up a class's state and behavior. 
+
+An **attribute** is just a specific type of **variable**: one defined within a class. Similarly a **method** is just a specific type of function **function**: one defined within a class. 
```

Comment:
```suggestion
An **attribute** is just a specific type of **variable**: one defined within a class. Similarly a **method** is just a specific type of **function**: one defined within a class.
```

I know that Simon's not a fan of 'just', so maybe drop those too? But I'll let them chime in if they want to.